### PR TITLE
add option to separate title from artist into two lines

### PR DESCRIPTION
**Pull request by 2LuckyLuke**
Hey I really like your applet, but I thought it was missing the option to separate the title and artist into two lines. I haven't really taken part in FOSS so far, so please excuse if this is not the standard way to do things. 

If you have any comments on this feature please let me know.
I would be happy to contribute to your applet.

Best Luke

### CODE REVIEW

File: src/contents/ui/main.qml [hunk 1]
```diff
@@ -44,13 +44,39 @@ Item {
...
+                    id: column
+                    spacing: 0
+                    anchors.fill: parent
+                    ScrollingText {
+                        overflowBehaviour: plasmoid.configuration.textScrollingBehaviour
+                        font.bold: true
+                        speed: plasmoid.configuration.textScrollingSpeed
+                        maxWidth: plasmoid.configuration.maxSongWidthInPanel * units.devicePixelRatio
+                        text: player.title
+                    }
```
Comment (ccatterina) [inline src/contents/ui/main.qml:62]:
```suggestion
                    }
```
Can you remove this and the other trailing spaces? If you use vs-code you can highlight them with `Trailing Spaces` plugin

**Review by ccatterina — COMMENTED** [review]
Hi @2LuckyLuke 

Thank you so much for your contribution to the project, you did a really good job!

I just leaved some comments for a few minor adjustments.

File: src/contents/ui/main.qml (hunk 1 above)
Comment (2LuckyLuke) [reply src/contents/ui/main.qml]: Done. If you would like to have anything else changed just let me know :)

Also happy new year 🎉

**Comment by ccatterina** [comment]
Great job! 🚀 

I will publish the new release into the store this evening

**Pull request merged**